src/CompilerWarnings.cmake

```diff
@@ -68,6 +63,13 @@ function(set_project_warnings project_name WARNINGS_AS_ERRORS MSVC_WARNINGS CLAN
     )
   endif()
 
+  message(WARNING "${WARNINGS_AS_ERRORS}")
```

Comment:
A better message for this would be good

Reply:
I would better remove this message, or not?